src/cascadia/TerminalSettingsModel/ActionAndArgs.cpp

```diff
@@ -51,6 +51,7 @@ static constexpr std::string_view LegacyToggleRetroEffectKey{ "toggleRetroEffect
 static constexpr std::string_view ToggleShaderEffectsKey{ "toggleShaderEffects" };
 static constexpr std::string_view MoveTabKey{ "moveTab" };
 static constexpr std::string_view BreakIntoDebuggerKey{ "breakIntoDebugger" };
+static constexpr std::string_view TogglePaneReadOnlyKey{ "togglePaneReadOnly" };
```

Comment:
```suggestion
static constexpr std::string_view TogglePaneReadOnlyKey{ "toggleReadOnlyMode" };
```
Maybe? Curious what @cinnamon-msft thinks.

Reply:
I definitely prefer `toggleReadOnlyMode`